agentMET4FOF/ml_uncertainty/bnn.py

```diff
@@ -1,54 +1,11 @@
-from agentMET4FOF.agents import AgentMET4FOF
-from examples.ZEMA_EMC.zema_feature_extract import FFT_BFC, Pearson_FeatureSelection
-
-import numpy as np
-import time
-
-from matplotlib import pyplot as plt
-from sklearn.discriminant_analysis import LinearDiscriminantAnalysis
-from sklearn import linear_model
-from sklearn.ensemble import RandomForestRegressor
-from sklearn.metrics import mean_squared_error
-from sklearn.model_selection import train_test_split
-
-from sklearn.metrics import f1_score
-from sklearn.metrics import confusion_matrix
-
-from scipy.stats import skew
-from scipy.stats import kurtosis
-from scipy.fftpack import fft
-
-import pandas as pd
+from agentMET4FOF.ml_uncertainty.bnn import *
```

Comment:
This is the actual origin of the strange behaviour of the import optimization . It is clearly a bug in PyCharm, but this statement on its own is kind of irritating. This basically means "import everything from in here" but why should we do that? PyCharm's import optimization resolves this and notices, that there is acutally everything present, if this import is performed and then deletes all other imports already present, but this of course recursively deletes the actual imports.